assets/css/_docs.scss

```diff
@@ -160,7 +160,7 @@ html.docs {
     input {
       border: none;
       border-bottom: 2px solid $defaultLiteColor;
-      font-size: 1.3rem;
+      font-size: 16px;
```

Comment:
Since we're using rem with a base of 10 you can make this 16px by setting font-zie to 1.6rem;